samples/snippets/src/test/java/com/example/bigquerydatatransfer/ListTransferConfigsIT.java

```diff
@@ -0,0 +1,119 @@
+import com.google.cloud.bigquery.datatransfer.v1.TransferConfig;
+import com.google.protobuf.Struct;
+import com.google.protobuf.Value;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ListTransferConfigsIT {
+
+  private BigQuery bigquery;
+  private ByteArrayOutputStream bout;
+  private String name;
+  private String displayName;
+  private String datasetName;
+  private PrintStream out;
+  private PrintStream realOut;
+
+  private static final String PROJECT_ID = requireEnvVar("GOOGLE_CLOUD_PROJECT");
+
+  private static String requireEnvVar(String varName) {
+    String value = System.getenv(varName);
+    assertNotNull(
+        "Environment variable " + varName + " is required to perform these tests.",
+        System.getenv(varName));
+    return value;
+  }
+
+  @BeforeClass
+  public static void checkRequirements() {
+    requireEnvVar("GOOGLE_CLOUD_PROJECT");
+  }
+
+  @Before
+  public void setUp() {
+    realOut = System.out;
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+
+    displayName = "MY_SCHEDULE_NAME_TEST_" + UUID.randomUUID().toString().substring(0, 8);
+    datasetName = "MY_DATASET_NAME_TEST_" + UUID.randomUUID().toString().substring(0, 8);
+    // create a temporary dataset
+    bigquery = BigQueryOptions.getDefaultInstance().getService();
+    bigquery.create(DatasetInfo.of(datasetName));
+
+    // create a scheduled query
+    String query =
+        "SELECT CURRENT_TIMESTAMP() as current_time, @run_time as intended_run_time, "
+            + "@run_date as intended_run_date, 17 as some_integer";
+    String destinationTableName =
+        "MY_DESTINATION_TABLE_" + UUID.randomUUID().toString().substring(0, 8) + "_{run_date}";
+    Map<String, Value> params = new HashMap<>();
+    params.put("query", Value.newBuilder().setStringValue(query).build());
+    params.put(
+        "destination_table_name_template",
+        Value.newBuilder().setStringValue(destinationTableName).build());
+    params.put("write_disposition", Value.newBuilder().setStringValue("WRITE_TRUNCATE").build());
+    params.put("partitioning_field", Value.newBuilder().setStringValue("").build());
+    TransferConfig transferConfig =
+        TransferConfig.newBuilder()
+            .setDestinationDatasetId(datasetName)
+            .setDisplayName(displayName)
+            .setDataSourceId("scheduled_query")
+            .setParams(Struct.newBuilder().putAllFields(params).build())
+            .setSchedule("every 24 hours")
+            .build();
+    CreateScheduledQuery.createScheduledQuery(PROJECT_ID, transferConfig);
+    String result = bout.toString();
+    name = result.substring(result.indexOf(".") + 1);
+
+    bout = new ByteArrayOutputStream();
```

Comment:
I don't think this is required anymore -- sorry for not seeing it earlier

Reply:
Done. Thank you!